docs/dev-client-gameplay.md

```diff
@@ -1,161 +1,333 @@
 ---
 id: dev-client-gameplay
-title: Hubs Client development Gameplay
+title: Core Concepts for Gameplay Code
 ---
 
-- [Intro](#org71ab195)
-- [Entities, components, systems.](#org739a535)
-  - [`bitECS`](#org0bfe8ab)
-  - [Disclaimer](#orgff3efea)
-- [Writing systems](#org751fbe1)
-  - [Systems are functions](#org775dcd0)
-  - [The game loop](#orga589afa)
-  - [Queries are useful](#org1d5d556)
-  - [Replacing `async` functions with `coroutines`](#org2233e62)
-- [Writing components](#org7331485)
-  - [Defining components](#org18a6d0b)
-  - [Data types](#org3db6503)
-  - [Avoid holding references](#org589f3d9)
-  - [Entity ID&rsquo;s are recycled](#org31d40f6)
-  - [String data](#orgdf10967)
-  - [Flags](#org70bd5dd)
-  - [Tag components](#org58d595d)
-  - [The escape hatch](#orga5547d0)
-  - [Associating entities with `Object3D` s](#orgf29a631)
-  - [Avoid duplicating state](#orgfe9fb09)
-- [Adding entities](#orgbad6d63)
-  - [Entity basics](#org9d285be)
-  - [Creating `EntityDef` s](#orgdd790f5)
-  - [Creating model files](#org156c909)
-  - [Entity creation is synchronous](#org2319bae)
-  - [Inflation](#org628683b)
-    - [What does `renderAsEntity` do?](#org20a38aa)
-    - [`Inflator` s](#org357d40d)
-    - [Default inflators](#orga810c47)
-    - [Associating `Object3D` s (`eid2obj`)](#org4f9fcd3)
-    - [Loading model files](#org7a41123)
-    - [Common inflators, `jsxInflators`, and `gltfInflators`](#org8ca2b33)
-    - [Entity `Ref` s and `__mhc_link_type` : `"node"`](#orgfcfa191)
-    - [Associating `Material` s (`eid2mat`)](#org7fe068b)
-- [Custom clients and addons](#org1a8110a)
-  - [Addons are not ready yet (February 2023)](#org2b06d43)
-  - [Creating an add-on](#org35960ac)
-  - [`preload`](#orge79e4e1)
-  - [Inserting prefabs](#org1d14013)
-  - [Inserting inflators](#org86ae847)
-  - [Inserting system calls](#org354b138)
-  - [Handling interactions](#org306c661)
-  - [Handling networking](#org19b999f)
+## Intro

```

Comment:
Thanks for the suggestion. I have been thinking about it but don't have a conclution yet because the "Add-ons` section can be too huge. Let me merge this PR as is for now, and let's think of making a follow up PR if needed.